src/behavior/src/A32NX_Interior_Misc.xml

```diff
@@ -166,20 +166,29 @@
                    at least currently, the autobrake system is activating and deactivating during bounced landings.
                    CONSIDER: If the autobrake system is rewritten to accurately simulate the real-life system, can we remove the delay?
                    -->
-            <Condition Check="AUTOBRAKE_LEVEL" Match="0">
-                <False>
+            <Condition Check="AUTOBRAKE_LEVEL" Match="1">
+                <True>
```

Comment:
Heh, so the condition for the update code is a bit misleading. As this template is used by all the three buttons, it would cause the update to run three times, this condition prevents that as the code itself still affects all levels potentially. 

The condition you mentioned is only important when spawning on the runway (makes sure the initial value does not get overriden) but the assignment is useless. I removed it.